backend/experiment/actions/trial.py

```diff
@@ -1,50 +1,86 @@
 from django.utils.translation import gettext_lazy as _
+from typing import Optional, TypedDict
 
 from .base_action import BaseAction
 from experiment.actions.form import Form
 from experiment.actions.playback import Playback
 from .frontend_style import FrontendStyle
 
 
+class Config(TypedDict):
+    """
+    Configuration for the Trial action.
+
+    Args:
+      - response_time (int): how long to wait until stopping the player
+      - auto_advance (bool): proceed automatically after stopping
+      - listen_first (bool): block form submission until playback ends
+      - show_continue_button (bool): display a 'Continue' button
+      - continue_label (str): label for the continue button
+    """
+
+    response_time: int
+    auto_advance: bool
+    listen_first: bool
+    show_continue_button: bool
+    continue_label: str
+
+
 class Trial(BaseAction):  # pylint: disable=too-few-public-methods
     """
-    A view that may include Playback and/or a Form
-    Relates to client component: Trial.js
+    A view that may include Playback and/or a Form to be displayed to the participant.
+
+    Args:
+        playback (Optional[Playback]): Player(s) to be displayed in this view (e.g. audio, video, image)
+        html (Optional[str]): HTML to be displayed in this view
+        feedback_form (Optional[Form]): array of form elements
+        title (Optional(str)): page title - defaults to empty
+        config (Optional[Config]): dictionary with following settings
```

Comment:
following where?

Reply:
Ah yes, I removed those as they are explained in the `Config` class already. I'll see what I can do

Reply:
https://github.com/Amsterdam-Music-Lab/MUSCLE/pull/1451/commits/d39686cc2c2a85c9fa542cd48e41546766de9714